docs/streams/stream-basics/subscriptions/exclusive-example.md

```diff
@@ -0,0 +1,402 @@
+    if has_stream:
+        print("This stream already exists!")
+        print(f"Existing Producer = c8globals.${stream_name}")
+    else:
+        print("\nCreating global stream...")
+        stream_info = client.create_stream(stream_name, False)
+        print(f"New Producer = ${stream_info['stream-id']}")
+
+
+""" Create producer and send data through a stream """
+def create_producer():
+    create_stream()
+
+    producer = client.create_stream_producer(stream_name, local=False)
+    while True:
+        message = f"Hello Macrometa Stream! Here is your random message number {random.randint(1, 100)}"
+        producer.send(message)
+        time.sleep(1)
+
+
+create_producer()
+```
+
+</TabItem>
+</Tabs>
+
+## Code example for consumer 1
+
+<Tabs groupId="modify-single">
+<TabItem value="javascript" label=" JavaScript">
+
+- Step 1. [Install the SDK](../../../sdks/install-sdks.md).
+- Step 2. Change `BASE_URL` if necessary and insert `apiKey`.
+- Step 3. Create an instance of the jsC8.
+- Step 4. Request `stream` object.
+- Step 5. Request One Time Password and create consumer.
+
+```js
+const jsc8 = require("jsc8");
+
+const BASE_URL = "https://play.paas.macrometa.io/"
+
+client = new jsc8({
+    url: BASE_URL,
+    apiKey: "xxxxxx",
+    fabricName: "_system",
+});
+
+const streamName = "streamQuickstart";
+const subscriptionName = "consumer-subscription"
+
+async function createStream() {
+    if (await client.hasStream(streamName, false)) {
+      console.log("This stream already exists!");
+      console.log(`Existing Consumer = c8globals.${streamName}`);
+    } else {
+      console.log("\nCreating global stream...");
+      // To create a global stream, set the second parameter to false
+      // There is an option to create a local stream, which is only accessible within the region
+      const streamInfo = await client.createStream(streamName, false);
+      console.log(`New Consumer = ${streamInfo.result["stream-id"]}`);
+    }
+  }
+
+async function consumer() {
+  try {
+    await console.log("\nConnecting consumer to global stream...");
+    // Create stream only if stream does not exist
+    createStream();
+    // Request stream object
+    const stream = client.stream(streamName, false);
+    // Request One Time Password
+    const consumerOTP = await stream.getOtp();
+    // Create consumer
+    const consumer = await stream.consumer(subscriptionName, BASE_URL.replace("https://",""), {
+      otp: consumerOTP
+    });
+    // Run consumer - open connection to server
+    consumer.on("message", (msg) => {
+      const { payload, messageId } = JSON.parse(msg);
+      // Received message payload
+      console.log(Buffer.from(payload, "base64").toString("ascii"));
+      // Send message acknowledgement
+      consumer.send(JSON.stringify({ messageId }));
+    });
+  } catch (e) {
+    await console.log("Could not receive messages " + e);
+  }
+}
+
+consumer();
+```
+</TabItem>
+<TabItem value="python" label="Python">
```

Comment:
```suggestion
<TabItem value="python" label="Python SDK">
```

Reply:
@diana-macrometa Sorry, I totally missed the new "Example" docs under the Subscriptions sub-folder.  Will revise those and fix the formatting.